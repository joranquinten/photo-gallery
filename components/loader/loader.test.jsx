import React from 'react';
import { shallow } from 'enzyme';

import Loader from './loader';

describe('Loader', () => {
  it('should render without crashing', () => {
    expect(shallow(<Loader>Child</Loader>)).toMatchSnapshot();

    expect(shallow(<Loader />)).toMatchSnapshot();
  });
});
