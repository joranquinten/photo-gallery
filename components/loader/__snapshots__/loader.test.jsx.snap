// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render without crashing 1`] = `
<div
  className="jsx-3132450403 loader"
>
  Child
  <JSXStyle
    css=""
    styleId="3132450403"
  />
</div>
`;

exports[`Loader should render without crashing 2`] = `
<div
  className="jsx-3132450403 loader"
>
  Loading...
  <JSXStyle
    css=""
    styleId="3132450403"
  />
</div>
`;
